### PR TITLE
GTKorvo 

### PR DESCRIPTION
**Pull request by kshitij-v-mehta**
Set of libraries for event transport middleware layer communication from the GTKorvo group.

### CODE REVIEW

File: var/spack/repos/builtin/packages/gtkorvo-cercs-env/package.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class GtkorvoCercsEnv(CMakePackage):
+    """A utility library used by some GTkorvo packages.
+    """
+
+    homepage = "http://www.example.com"
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/gtkorvo-cercs-env/package.py]: Needs a homepage.

File: var/spack/repos/builtin/packages/gtkorvo-enet/package.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    """
+
+    homepage = "http://www.github.com/GTkorvo/enet"
+    url = "https://github.com/GTkorvo/enet/archive/v1.3.13.tar.gz"
+
+    version('1.3.13', '3490f924a4d421e4832e45850e6ec142')
+
+    def configure_args(self):
+        args = []
+        return args
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/gtkorvo-enet/package.py]: You can get rid of this is you're not going to use it.

**Comment by adamjstewart** [comment]
I think you removed the wrong `configure_args`.

**Comment by kshitij-v-mehta** [comment]
Hi @tgamblin , I have addressed Adam's comments, could you let me know if this PR needs further review? I don't mean to rush you, I was just wondering if I could do something to help push the PR along (a check seems to have failed, but I am not sure what the failure is).

**Comment by adamjstewart** [comment]
> a check seems to have failed, but I am not sure what the failure is

You can ignore that. It just means that the overall unit test coverage in Spack has increased since you branched off of develop. That's a good thing!

To clarify, we pretty much ignore codecov pass/fail if you only edit package files. As long as Travis passes, you're usually good to go.

Ping @tgamblin

**Pull request merged**